src/sentry/tasks/relay.py

```diff
@@ -32,6 +32,9 @@ def build_project_config(public_key=None, **kwargs):
 
     Do not invoke this task directly, instead use :func:`schedule_build_project_config`.
     """
+    with configure_scope() as scope:
+        scope.set_tag("project.public_key", public_key)
```

Comment:
nit: There's already a `set_tag("public_key", ...)` in this file, should we use the same tag name here?